flink/metadata.csv

```diff
@@ -0,0 +1,89 @@
+flink.task.Shuffle.Netty.Input.Buffers.inPoolUsage,gauge,,,,An estimate of the input buffers usage,0,flink,task input buffers used
+flink.task.Shuffle.Netty.Output.Buffers.outputQueueLength,gauge,,,,The number of queued output buffers,0,flink,task output buffers queued
+flink.task.Shuffle.Netty.Output.Buffers.outPoolUsage,gauge,,,,An estimate of the output buffers usage,0,flink,task output buffers used
+flink.task.Shuffle.Netty.Input.numBytesInLocal,count,,byte,,The total number of bytes this task has read from a local source,0,flink,task bytes read local
+flink.task.Shuffle.Netty.Input.numBytesInLocalPerSecond,gauge,,byte,second,The number of bytes this task reads from a local source per second,0,flink,task bytes read local per second
+flink.task.Shuffle.Netty.Input.numBytesInRemote,count,,byte,,The total number of bytes this task has read from a remote source,0,flink,task bytes read remote
+flink.task.Shuffle.Netty.Input.numBytesInRemotePerSecond,gauge,,byte,,The number of bytes this task reads from a remote source per second,0,flink,task bytes read remote per second
+flink.task.Shuffle.Netty.Input.numBuffersInLocal,count,,,,The total number of network buffers this task has read from a local source,0,flink,task buffers read local
+flink.task.Shuffle.Netty.Input.numBuffersInLocalPerSecond,gauge,,,second,The number of network buffers this task reads from a local source per second,0,flink,task buffers read local per second
+flink.task.Shuffle.Netty.Input.numBuffersInRemote,count,,,,The total number of network buffers this task has read from a remote source,0,flink,task buffers read remote
+flink.task.Shuffle.Netty.Input.numBuffersInRemotePerSecond,gauge,,,second,TThe number of network buffers this task reads from a remote source per second,0,flink,task buffers read remote per numBuffersInLocalPerSecond
+flink.jobmanager.numRegisteredTaskManagers,gauge,,,,The number of registered taskmanagers,0,flink,jobmanager registered taskmanagers
+flink.jobmanager.numRunningJobs,gauge,,,,The number of running jobs,0,flink,jobmanager running jobs
+flink.jobmanager.taskSlotsTotal,gauge,,,,The total number of task slots,0,flink,jobmanager task slots
+flink.jobmanager.job.restartingTime,gauge,,millisecond,,The time it took to restart the job or how long the current restart has been in progress,0,flink,jobmanager job restart time
+flink.jobmanager.job.uptime,gauge,,millisecond,,The time that the job has been running without interruption. Returns -1 for completed jobs,0,flink,jobmanager job uptime
+flink.jobmanager.job.downtime,gauge,,millisecond,,For jobs currently in a failing/recovering situation- the time elapsed during this outage. Returns 0 for running jobs and -1 for completed jobs,-1,flink,jobmanager job downtime
+flink.jobmanager.job.fullRestarts,gauge,,,,The total number of full restarts since this job was submitted. Attention: Since 1.9.2 this metric also includes fine-grained restarts,-1,flink,jobmanager job restarts
```

Comment:
Can we also add `numRestarts`. According to this doc, fullRestarts is deprecated https://ci.apache.org/projects/flink/flink-docs-stable/monitoring/metrics.html#availability

Reply:
Yes! It looks like it was just deprecated in 1.10, which came out really recently. I will double check the other metrics to see if anything else changed

Reply:
Updated! Looks like this was the only new deprecation.